Jenkinsfile-pr

```diff
@@ -23,7 +26,7 @@ pipeline {
     parameters {
         string(name: 'TEST_CASE', defaultValue: '*ST', description: 'maven parameter for executing specific tests')
         string(name: 'TEST_PROFILE', defaultValue: 'acceptance', description: 'maven parameter for executing specific test profile')
-        string(name: 'EXCLUDE', defaultValue: 'networkpolicies', description: 'maven parameter for exclude specific test tag')
+        string(name: 'EXCLUDE', defaultValue: ${DEFAULT_EXCLUDED_GROUPS}, description: 'maven parameter for exclude specific test tag')
```

Comment:
I think this will fail the build -> it should be like this: `"${DEFAULT_EXCLUDED_GROUPS}"` - or am I wrong?